src/Report/EnabledMembers.php

```diff
@@ -0,0 +1,167 @@
+                $registeredMethodNames = array_map(function ($methodClass) use ($methods) {
+                    return $methods[$methodClass];
+                }, $registeredMethods);
+                $registeredMethodList = implode(', ', $registeredMethodNames);
+                $methodCount = count($registeredMethods);
+                $member->methodCount = $methodCount ?: '';
+                $member->registeredMethods = $registeredMethodList;
+                $member->defaultMethodName = $methods[$defaultMethodClassName] ?? '';
+            }
+            $member->skippedRegistration = $member->HasSkippedMFARegistration
+                ? $member->dbObject('HasSkippedMFARegistration')->Nice()
+                : '';
+        });
+        $filteredList = ArrayList::create($members);
+        if (isset($countFilter)) {
+            $filteredList = $filteredList->filter('methodCount', $countFilter);
+        }
+        return $filteredList;
+    }
+
+    public function columns()
+    {
+        return singleton(Member::class)->summaryFields() + [
+            'methodCount' => _t(__CLASS__ . '.COLUMN_METHOD_COUNT', 'Number of registered methods'),
+            'registeredMethods' => _t(__CLASS__ . '.COLUMN_METHODS_REGISTERED', 'Registered methods'),
+            'defaultMethodName' => _t(__CLASS__ . '.COLUMN_METHOD_DEFAULT', 'Default method'),
+            'skippedRegistration' => _t(__CLASS__ . '.COLUMN_SKIPPED_REGISTRATION', 'Skipped registration'),
+        ];
+    }
+
+    public function parameterFields()
+    {
+        $methods = $this->getMethodClassToTitleMapping();
+        unset($methods[get_class(Injector::inst()->get(MethodRegistry::class)->getBackupMethod())]);
+        return FieldList::create(
+            TextField::create(
+                'Member',
+                singleton(Member::class)->i18n_singular_name()
+            )->setDescription(_t(
+                __CLASS__ . '.FILTER_MEMBER_DESCRIPTION',
+                'Firstname, Surname, Email partial match search'
+            )),
+            NumericField::create(
+                'Count',
+                _t(__CLASS__ . '.COLUMN_METHOD_COUNT', 'Number of registered methods')
+            )->setHTML5(true),
+            DropdownField::create(
+                'Methods',
+                _t(__CLASS__ . '.COLUMN_METHODS_REGISTERED', 'Registered methods'),
+                $methods,
+                ''
+            )->setHasEmptyDefault(true),
+            CheckboxField::create(
+                'Skipped',
+                _t(__CLASS__ . '.COLUMN_SKIPPED_REGISTRATION', 'Skipped registration')
+            )
+        );
+    }
+
+    protected function getMethodClassToTitleMapping()
+    {
+        $methods = array_flip(ClassInfo::implementorsOf(MethodInterface::class));
+        array_walk($methods, function (&$name, $class) {
+            $name = Injector::inst()->get($class)->getName();
+        });
+        return $methods;
```

Comment:
I think using `array_walk` with a reference argument here is a little convoluted. My suggestion is to use `array_map` instead:

```php
return array_map(function($class) {
    return Injector::inst()->get($class)->getName();
}
```

Although by using `MethodRegistry` you already have instances of the methods so it'll just be `return $method->getName()` instead of using `Injector`.

Reply:
Fair, a natural evolution from a previous version that was using both name and class.